canonical-web-and-design/snapcraft.io
# Distro page snap details

Fixes #1951

Updates styling of snap details section of distro page.

### QA
- ./run or https://snapcraft-io-canonical-web-and-design-pr-1957.run.demo.haus/
- go to distro page of any snap (/install/SNAPNAME/arch or /instal/SNAPNAME/debian are available)
- snap details section should be collapsed by default
- click 'See more' to expand it
- check different snaps to see if various size descriptions, images, video, etc work

<img width="1359" alt="Screenshot 2019-06-04 at 16 20 40" src="https://user-images.githubusercontent.com/83575/58886819-cfe3f780-86e4-11e9-8970-8a32ef82f8dd.png">
